test/OutputCacheProviderFunctionalTests/RedisOutputCacheProvider.FunctionalTests.csproj

```diff
@@ -16,19 +16,19 @@
     <ProjectReference Include="..\..\src\OutputCacheProvider\RedisOutputCacheProvider.csproj" />
```

Comment:
Changes to the OutputCacheProvider should be in a separate PR if possible

Reply:
Changed